addons/account/static/src/xml/account_reconciliation.xml

```diff
@@ -61,6 +61,7 @@
                     rel="do_action"
                     data-action_name="Unreconciled Entries"
                     data-model="account.move.line"
+                    data-domain="[('parent_state', '=', 'posted')]"
```

Comment:
Maybe should be something like
```suggestion
                    data-domain="['|', ('matched_debit_ids', '!=', []), ('matched_credit_ids', '!=', [])]"
```
as per https://github.com/odoo/odoo/pull/92666 :)